mne/filter.py

```diff
@@ -626,6 +626,10 @@ def band_pass_filter(x, Fs, Fp1, Fp2, filter_length='10s',
     xf : array
         x filtered.
 
+    See Also
+    --------
+    low_pass_filter, high_pass_filter
```

Comment:
doesn't sound very related to me ... so I'll skip this for now. But thanks for the suggestion!